calico_node/calico-bgp-daemon/main.go

```diff
@@ -45,13 +46,31 @@ const (
 	ETCD_CERT_FILE    = "ETCD_CERT_FILE"
```

Comment:
On a second thought, you can leave this as it is since it might be better to have these names uniform across the repo also, Python code has it that way.